cmd/drone-docker/main.go

```diff
@@ -247,7 +262,10 @@ func main() {
 
 func run(c *cli.Context) error {
 	plugin := docker.Plugin{
-		Dryrun:  c.Bool("dry-run"),
+		Dryrun:            c.Bool("dry-run"),
+		Loadimage:         c.Bool("load-image"),
+		Loadedimagetag:    c.String("loaded-image-tag"),
```

Comment:
It is a duplication of string... the same string as in line 247.
Isn't there a way to deduplicate?

Reply:
Yup, they're all duplicated - however that's the plugin convention + it's contained to a single file